### PR TITLE
db/navigator-colors

### PR DESCRIPTION
**Pull request by bre1470**
I struggle with the shapes of the handle elements in navigator, because these are SVGs. And the reposition of the scrollbar to the top calls also for a hack.

### CODE REVIEW

**Comment by pawelfus** [comment]
> I struggle with the shapes of the handle elements in navigator, because these are SVGs

Created simple arrows for that: https://jsfiddle.net/BlackLabel/xqew7o6p/

> And the reposition of the scrollbar to the top calls also for a hack.

Please, skip that.

**Review by pawelfus — CHANGES_REQUESTED** [review]
Looks good! Just some nitpicking:

- I don't believe I'm the one asking this.. 😅 but could you please move the right arrowhead by 1-2px to the right? The same distance from the mask border as the left one: 
image
- could you change the scrollbar's track background? As for the scrollbar buttons, let's keep them as they are or use the same colors as mask arrows. What do you think @nasin ? Right now it's not possible to disable them

File: samples/dashboard/demos/climate/demo.css [hunk 1]
```diff
@@ -18,38 +64,86 @@
...
 
+/* *
+ *
+ *  KPI
+ *
+ * */
+
+.hd-cell.hd-cell-state-hover:hover > .hd-component,
+.hd-cell.hd-cell-state-active > .hd-component {
+    background-color: #46475d;
```
Comment (nasin) [inline samples/dashboard/demos/climate/demo.css:75]: I was using this as the highlight color for table cells: #5352BE

**Review by nasin — APPROVED** [review]
All of this looks good. I just noted that the highlight color for the table rows is different from what I have in Figma.

**Review by sebastianbochan — APPROVED** [review]

**Comment by pawelfus** [comment]
I don't see my requested changes..?

**Pull request merged**